docs/modules/operation/pages/topology/asset-topology-provider/node-filtering.adoc

```diff
@@ -0,0 +1,49 @@
+[[node-filtering]]
+= Node Filtering
+
+In many cases, it is desirable to control which nodes a topology includes or excludes.
+For instance, it is useful to be able to generate customized topologies for specific customers that include only regions/buildings, and so on, relevant to their filtered node set.
+To this end, it is possible to define a node filter that chooses which nodes are included in a generated topology.
+
+Define filters using the same asset table keys that are available for the `assetLayers` field.
+
+[options="header"]
```

Comment:
we should redo the table using our new style (cells on a separate line):
|===
| Operation
| Definition
| Example

etc ...